src/libs/runtime-utils.js

```diff
@@ -77,14 +87,33 @@ export const getValidGpuTypes = (numCpus, mem, zone) => {
   return validGpuTypes || { name: '?', type: '?', numGpus: '?', maxNumCpus: '?', maxMem: '?', price: NaN, preemptiblePrice: NaN }
 }
 
-const gpuCost = (gpuType, numGpus) => _.find({ type: gpuType, numGpus }, gpuTypes)?.price || NaN
-
 const dataprocCost = (machineType, numInstances) => {
   const { cpu: cpuPrice } = findMachineType(machineType)
 
   return cpuPrice * numInstances * dataprocCpuPrice
 }
 
+const getHourlyCostForMachineType = (machineTypeName, region, isPreemptible) => {
+  const { cpu, memory } = _.find({ name: machineTypeName }, machineTypes)
+  const { n1HourlyCpuPrice, preemptibleN1HourlyCpuPrice, n1HourlyGBRamPrice, preemptibleN1HourlyGBRamPrice } = _.find({ name: _.toUpper(region) },
+    regionToPrices)
+  return !isPreemptible ?
+    (cpu * n1HourlyCpuPrice) + (memory * n1HourlyGBRamPrice) :
+    (cpu * preemptibleN1HourlyCpuPrice) + (memory * preemptibleN1HourlyGBRamPrice)
```

Comment:
Could you flip this? We like positive case first, when reasonably possible:
```suggestion
  return isPreemptible ?
    (cpu * preemptibleN1HourlyCpuPrice) + (memory * preemptibleN1HourlyGBRamPrice) :
    (cpu * n1HourlyCpuPrice) + (memory * n1HourlyGBRamPrice)
```

Reply:
The reason for having the negative case first was because `isPreemptible` will be `false` most of the time this ternary is executed and I wanted the evaluation to be able to stop sooner. I understand that it's probably premature optimization and don't feel strongly about it.

Reply:
All other things being equal, then, I'd prefer to avoid the premature optimization.

Reply:
Done.